content/ember-data/v3/default-serializer.md

```diff
@@ -17,21 +19,21 @@ These classes specified the following `defaultSerializer`
 - `RESTAdapter`: `-rest` (`@ember-data/serializer/rest`)
 - `JSONAPIAdapter`: `-json-api` (`@ember-data/serializer/json-api`)
 
-If a per-type adapter exists for the modelName triggering this deprecation, the easiest resolution is to add a per-type serializer.
-
-If the application adapter is triggering this deprecation, then an application serializer should be added.
-
-More information on adding these serializers can be found in "clearing these deprecations" below.
+### Clearing these deprecations
 
-### -default serializer fallback in store.serializerFor
+If all the adapters in your app are the same kind (such as JSONAPI, REST, or JSON), you should create an
+application serializer to match.
 
-Previously, when no type-specific serializer, application serializer, or adapter defaultSerializer had been defined by the app, the `-default` serializer would be used which defaulted to the `JSONSerializer`. This behavior is deprecated in favor of explicitly defining an application or type-specific serializer as described below.
+For example, if you use only JSONAPI adapters, creating the file below in `app/serializers/application.js`
+will resolve the deprecation:
 
-## clearing these deprecations
+```javascript
+// app/serializers/application.js
 
-More information about custom serializers can be found in the [Serializer API Docs](https://api.emberjs.com/ember-data/release/modules/@ember-data%2Fserializer) or on the [ember.js/guides](https://guides.emberjs.com/release/models/customizing-serializers/#toc_customizing-serializers)
+export { default } from '@ember-data/serializer/json-api';
+```
 
-If a specific model type requires custom serialization, a type-specific serializer can be created. A single `application` serializer can be used for any model types not requiring custom serialization. To define a type-specific serializer, create an `app/serializers/[type].js` with the following:
+If your app uses different adapter types for different models, you should make one serializer for each model type. For example, if a certain model uses the `RESTAdapter`, create an `app/serializers/[model-type].js` file with the following:
```

Comment:
What if you give a concrete model name as an example and use the `data-filename` facility to show the path?